integration-tests/src/test/java/oracle/kubernetes/operator/ItSitConfigDomainInPV.java

```diff
@@ -0,0 +1,323 @@
+// Copyright (c) 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.logging.Level;
+
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.LoggerHelper;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * JUnit test class used for testing configuration override use cases for domain in pv WLS.
+ */
+public class ItSitConfigDomainInPV extends SitConfig {
+  private static String testClassName;
+  private static int testNumber;
+  private static Operator operator1;
+  private static Domain domain;
+  private static String sitconfigTmpDir = "";
+  private static String mysqltmpDir = "";
+  private static String configOverrideDir = "";
+  private static String mysqlYamlFile = "";
+  private static String domainNS;
+  private static String testprefix = "sitconfigdomaininpv";
+  private static String mysqldbport;
+  private static String ADMINPODNAME;
+  private static String JDBC_URL;
+  private static StringBuffer namespaceList;
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
```

Comment:
same comments as for the other ItSitConfig class.

Reply:
fixed